istioctl/cmd/istioctl/rbac.go

```diff
@@ -57,23 +47,41 @@ istioctl experimental rbac can -u test GET rating /v1/health
 istioctl experimental rbac can -s service=product-page POST rating /data -a version=dev`,
 		Args: cobra.ExactArgs(3),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			action.method = args[0]
-			action.service = args[1]
-			action.path = args[2]
-			action.namespace = namespace
+			action.Method = args[0]
+			action.Service = args[1]
+			action.Path = args[2]
+			if namespace == "" {
+				action.Namespace = "default"
+			} else {
+				action.Namespace = namespace
+			}
 
-			// TODO(yangminzhu): Implement the query logic.
-			return fmt.Errorf("not implemented.\nsubject: %+v and action: %+v", subject, action)
+			rbacStore, err := newRbacStore()
```

Comment:
Do we construct RbacStore every time a request comes? Can we construct it once and keep it in memory to serve requests? That is the model Mixer RBAC adapter currently has.